app/models/user.rb

```diff
@@ -57,6 +57,7 @@ class User < ApplicationRecord
   has_many :availabilities, dependent: :destroy
   validates :cell, presence: true
   validates :email, presence: true
-
+  validates :email, uniqueness: true
```

Comment:
Add tests for this in the model specs

Reply:
Bump

Reply:
would this be the same as the test case for Username { SecureRandom.alphanumeric(8) }? but appending the email suffix (e.x. @gmail.com)